server/container/filesystem-api/src/test/java/org/apache/james/filesystem/api/AbstractFileSystemTest.java

```diff
@@ -54,24 +62,47 @@
 
     protected FileSystem fileSystem;
     
-    private Service httpServer;
+    private Server httpServer;
     private File rootDirectory;
     
     protected abstract FileSystem buildFileSystem(String configurationRootDirectory);
 
     @Before
     public void setUp() throws Exception {
-        httpServer = Service.ignite().port(RANDOM_PORT);
-        httpServer.get("/", (Request request, Response response) -> "content");
-        httpServer.awaitInitialization();
-        
+        httpServer = new Server(RANDOM_PORT);
```

Comment:
you should have a look at JettyHttpServerTest, you'll find that you don't need that much boilerplate (: